components/ProfilePage/SmartWalletContents.tsx

```diff
@@ -4,14 +4,28 @@ import { useProfileProvider } from "../../providers/ProfileContext"
 import getSmartWallet from "../../lib/getSmartWallet"
 import getProfileFormattedCollection, { ALLNFTS } from "../../lib/getProfileFormattedCollection"
 import Media from "../../shared/Media"
+import getDefaultProvider from "../../lib/getDefaultProvider"
+import Deploy6551AndMintDNAButton from "./Deploy6551AndMintButton"
 
 const SmartWalletContents = () => {
   const { cre8orNumber } = useProfileProvider()
   const [ownedNfts, setOwnedNfts] = useState([])
-
+  const [hasSmartWallet, setHasSmartWallet] = useState(true)
+  const provider = getDefaultProvider(process.env.NEXT_PUBLIC_TESTNET ? 5 : 1)
   useEffect(() => {
     const init = async () => {
-      const smartWalletAddress = await getSmartWallet(cre8orNumber)
+      const smartWalletAddress = await getSmartWallet(85)
+      //
+      // Fetch the contract code
+      const code = await provider.getCode(smartWalletAddress)
+
+      if (code === "0x") {
+        console.log(`No contract deployed at address: ${smartWalletAddress}`)
```

Comment:
lint will complain about this.

Reply:
my guess is a lot of these hard coded values will be replaced after testing?

Reply:
removed dev logging ✅